package.json

```
@@ -22,6 +21,9 @@
     "lint": "eslint . --ext .ts,.js,.tsx,.jsx",
     "prepare": "husky install"
   },
+  "prisma": {
+    "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} ./prisma/seed.ts"
```

This is is already in the `ts-node` script above. Should be enough with 

```suggestion
    "seed": "ts-node ./prisma/seed.ts"
```

or possibly


```suggestion
    "seed": "yarn ts-node ./prisma/seed.ts"
```